### ISSUE TITLE
ask you questions about predictions.

### ISSUE THREAD

**Issue by BUPTHYP** [tkipf/gcn#122]
Hello Thomas , Thank you very much for replying to so many questions, your code has helped me a lot.I don't quite understand the predictions. i have 3 questions about it.
             
1. when we  train or  test our models, I found  ”output = model(features, adj)“  to be the same. Just  **F.nll_loss**  is different .I dont konw why we dont feed it to model ? 

2.Because of the above reasons, I am not very clear when my classification model is well trained. I saved the models, so if I have a new input, where should I feed?  Is the F.nll_loss? 

3.When we calculate the accuracy of the output，in **pygcn.utils.py  Line67**  
'''
def accuracy(output, labels):
    preds = output.max(1)[1].type_as(labels)
'''
the output is the log_softmax and the shape is (1000,7) . why  we choose the **output.max(1)[1]** not **output.max(1)[0]** ,Maybe it is a Too small problem········

But thank you very much for replying my questions .
